main/src/main/scala/sbt/Keys.scala

```diff
@@ -383,7 +383,7 @@ object Keys {
   val description = settingKey[String]("Project description.").withRank(BSetting)
   val homepage = settingKey[Option[URL]]("Project homepage.").withRank(BSetting)
   val startYear = settingKey[Option[Int]]("Year in which the project started.").withRank(BMinusSetting)
-  val licenses = settingKey[Seq[(String, URL)]]("Project licenses as (name, url) pairs.").withRank(BMinusSetting)
+  val licenses = settingKey[Seq[LicenseInfo]]("Project licenses.").withRank(BMinusSetting)
```

Comment:
While I'm +1 in principle, we should try to make the transition as smooth as possible to minimize the disruption cost. For example we should consider introducing a givens converter so sbt 1.x tuple would work out of box, and maybe show a warning.